cmd/createappledevtoken/createappledevtoken.go

```diff
@@ -109,6 +111,13 @@ func initWithEnv() error {
 		return fmt.Errorf("CreateAppleDevToken [Init Firestore]: %v", err)
 	}
 
+	// Initialize Sawmill
+	sawmillLogger, err := sawmill.InitClient(currentProject, os.Getenv("GCLOUD_CONFIG"), os.Getenv("ENVIRONMENT"), "CreateAppleDevToken")
+	if err != nil {
+		log.Printf("CreateAppleDevToken [Init Sawmill]: %v", err)
+	}
+
+	logger = sawmillLogger
 	firestoreClient = client
```

Comment:
[nit]

In the other function you set `logger = sawmillLogger` under the `firestoreClient`. Can you change that here?